.github/workflows/zip.yml

```diff
@@ -9,8 +13,29 @@ jobs:
   build:
     runs-on: macOS-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
What does v2 mean versus v1?  I haven't been able to find this in the docs?

Reply:
It's now the default for starting a new action - looks like Dec 13, 2019 it was released: https://github.com/actions/checkout/releases/tag/v2.0.0 

We should probably update the rest of the actions, but should do so in a separate PR.